Bottom: 81a62e35739b7d69fdd79afe1bdc155cecb020e9
Top:    55b60791f5749981a1f83503b197e128b35380ac
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 16:24:53 -0500

update worker configure like master


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index 75985b8a..4c29eb74 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -92,10 +92,10 @@ c['buildbotNetUsageData'] = 'basic'
 
 import openembedded
 reload(openembedded)
-c['workers'].extend(openembedded.workers)
-c['schedulers'].extend(openembedded.schedulers)
-c['builders'].extend(openembedded.builders)
-c['change_source'].extend(openembedded.change_source)
+c['workers'].extend(openembedded.WorkerConfig['workers'])
+c['schedulers'].extend(openembedded.WorkerConfig['schedulers'])
+c['builders'].extend(openembedded.WorkerConfig['builders'])
+c['change_source'].extend(openembedded.WorkerConfig['change_source'])
 
 import ptxdist
 reload(ptxdist)
diff --git a/docker/buildbot-master/buildbot/openembedded.py b/docker/buildbot-master/buildbot/openembedded.py
index 7e3643f2..8837daef 100755
--- a/docker/buildbot-master/buildbot/openembedded.py
+++ b/docker/buildbot-master/buildbot/openembedded.py
@@ -17,6 +17,7 @@ workers = [
 
 DEFAULT_REPO = 'git@git.novatech-llc.com:ntel/setup-scripts.git'
 
+
 # CHANGESOURCES
 change_source = [
     changes.GitPoller(
